.github/workflows/validate.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: verify staged ceremony changes
         run: |
           set -euo pipefail
-          export CHANGED=$(git diff --name-only HEAD^ HEAD ceremony/ | grep ".*staged.*" | head -1)
+          export CHANGED=$(git diff --name-only HEAD^ HEAD ceremony/20* | grep ".*staged.*" | head -1)
```

Comment:
what happens in the year 3000

Reply:
:D we celebrate our 980th anniversary with a potential outage